diff -burN bvi-1.3.2.orig/Makefile.in bvi-1.3.2/Makefile.in
--- bvi-1.3.2.orig/Makefile.in	2004-02-15 00:17:56.715259832 +0100
+++ bvi-1.3.2/Makefile.in	2004-02-15 01:07:21.133599744 +0100
@@ -26,6 +26,7 @@
 bindir = $(DESTDIR)@bindir@
 mandir = $(DESTDIR)@mandir@
 libdir = $(DESTDIR)@libdir@
+datadir = $(DESTDIR)@datadir@
 
 man1dir = $(mandir)/man1
 
@@ -58,7 +59,7 @@
 	$(CC) $(CFLAGS) $(DEFS) -c $<
 
 bmore.o: bmore.c
-	$(CC) $(CFLAGS) $(DEFS) -DHELPFILE=\"$(libdir)/bmore.help\" -c $<
+	$(CC) $(CFLAGS) $(DEFS) -DHELPFILE=\"$(datadir)/bvi/bmore.help\" -c $<
 
 
 install: all installdirs
@@ -66,14 +67,14 @@
 	@$(INSTALL_DATA) bvi.1 bmore.1 $(man1dir)
 	@$(INSTALL_PROGRAM) -s bvi $(bindir)
 	@$(INSTALL_PROGRAM) -s bmore $(bindir)
-	@$(INSTALL_DATA) bmore.help $(libdir)
+	@$(INSTALL_DATA) bmore.help $(datadir)/bvi
 	@if [ -f $(bindir)/bview ]; then rm $(bindir)/bview; fi
 	@if [ -f $(bindir)/bvedit ]; then rm $(bindir)/bvedit; fi
 	@ln $(bindir)/bvi $(bindir)/bview
 	@ln $(bindir)/bvi $(bindir)/bvedit
 
 installdirs:
-	$(SHELL) ${srcdir}/mkinstalldirs $(bindir) $(libdir) $(man1dir)
+	$(SHELL) ${srcdir}/mkinstalldirs $(bindir) $(datadir)/bvi $(man1dir)
 
 uninstall:
 	@echo "Uninstalling bvi"
